docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,474 @@
+# Nebula Graph CRUD
+
+This topic describes the basic CRUD operations in Nebula Graph.
+
+## Graph space and Nebula Graph schema
+
+A Nebula Graph instance consists of one or more graph spaces. Graph spaces are physically isolated from each other. You can use different graph spaces in the same instance to store different data sets.
+
+![Nebula Graph and graph spaces](https://docs-cdn.nebula-graph.com.cn/docs_2.0/3.quick-start/nebula_graph_instance_and_graph_spaces.png)
+
+Before inserting data into a graph space, you have to define the database schema. Nebula Graph schema is based on the following components.
+
+| Schema component | Description                                                                               |
+| ---------------- | ---------------------------------------------------------------------------------------   |
+| Vertex           | Represents an entity in the real world. One vertex can have multiple tags.                |
+| Tag              | The type of a vertex. It defines a group of properties that describes a type of vertices. |
+| Edge             | Represents a **directed** relationship between two vertices.                              |
+| Edge type        | The type of an edge. It defines a group of properties that describes a type of edges.     |
+
+For more information, see [Nebula Graph schema](TODO) [TODO].
+
+In this topic, we use the following data set to demonstrate basic CRUD operations.
+
+![The demo dataset](https://docs-cdn.nebula-graph.com.cn/docs_2.0/3.quick-start/demo_dataset_for_quick_start.png)
+
+## Check the machine status in the Nebula Graph cluster
+
+First of all, we recommend that you check the machine status in the Nebula Graph status to make sure all the storage servers (storaged) are connected to the metadata servers (metad). Run `SHOW HOSTS` as follows.
+
+```nGQL
+(root@nebula) [(none)]> SHOW HOSTS;
++-----------+-------+--------+--------------+---------------------+------------------------+
+| Host      | Port  | Status | Leader count | Leader distribution | Partition distribution |
++-----------+-------+--------+--------------+---------------------+------------------------+
+| storaged0 | 44500 | ONLINE | 0            | No valid partition  | No valid partition     |
++-----------+-------+--------+--------------+---------------------+------------------------+
+| storaged1 | 44500 | ONLINE | 0            | No valid partition  | No valid partition     |
++-----------+-------+--------+--------------+---------------------+------------------------+
+| storaged2 | 44500 | ONLINE | 0            | No valid partition  | No valid partition     |
++-----------+-------+--------+--------------+---------------------+------------------------+
+Got 3 rows (time spent 1775/2334 us)
```

Comment:
make sure there is some place explain `time spent 1775/2334 us`

Reply:
in FAQ in 1.0

Reply:
Added an FAQ at the end.